packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapController.m

```diff
@@ -547,28 +575,38 @@ - (void)mapView:(GMSMapView *)mapView idleAtCameraPosition:(GMSCameraPosition *)
 }
 
 - (BOOL)mapView:(GMSMapView *)mapView didTapMarker:(GMSMarker *)marker {
-  NSString *markerId = marker.userData[0];
-  return [self.markersController didTapMarkerWithIdentifier:markerId];
+  if ([marker.userData conformsToProtocol:@protocol(GMUCluster)]) {
+    GMUStaticCluster *cluster = marker.userData;
```

Comment:
should it be just `GMUCluster` since that's what you checked to conform to?

Reply:
Fixed the check